src/rx5808-pro-diversity/RX5808.cpp

```diff
@@ -0,0 +1,177 @@
+#include "RX5808.h"
+
+// PUBLIC FUNCTIONS
+RX5808::RX5808(void) :
+    minRSSI(DEFAULT_MIN_RSSI),
+    maxRSSI(DEFAULT_MAX_RSSI)
+{}
```

Comment:
You should be able to set these as defaults directly in the class definition instead rather than using initialisation lists.